rules/windows/registry/registry_set/registry_set_odbc_driver_registered.yml

```diff
@@ -0,0 +1,33 @@
+title: New ODBC Driver Registered
+id: 3390fbef-c98d-4bdd-a863-d65ed7c610dd
+status: experimental
+description: Detects the registration of a new ODBC driver.
+references:
+    - https://www.hexacorn.com/blog/2020/08/23/odbcconf-lolbin-trifecta/
+author: Nasreddine Bencherchali (Nextron Systems)
+date: 2023/05/23
+tags:
+    - attack.persistence
+logsource:
+    category: registry_set
+    product: windows
+detection:
+    selection:
+        EventType: SetValue
+        TargetObject|contains: '\SOFTWARE\ODBC\ODBCINST.INI\'
+        TargetObject|endswith: '\Driver'
+    filter_main_sqlserver:
+        TargetObject|contains: '\SQL Server\'
+        Details: '%WINDIR%\System32\SQLSRV32.dll'
+    filter_optional_office_access:
+        TargetObject|contains: 'I\Microsoft Access '
```

Comment:
Just making sure: The `I\...` is intentional?

Reply:
It doesn't hurt but It wasn't intentional. I removed it

Reply:
I couldn't image a path ending on capital I^^